tox.ini

```diff
@@ -1,7 +1,11 @@
 [tox]
-envlist = py27,py33
+envlist = django{18,19,110}
```

Comment:
I don't understand :/
We are building a python matrice in travis with python 2.7, 3.4, 3.5, and for each python version, we are running unittest through tox with Django versions: django 1.8, 1.9 and 1.10